ssr/src/page/wallet/mod.rs

```diff
@@ -27,7 +27,7 @@ fn ProfileGreeter(details: ProfileDetails) -> impl IntoView {
     // let (is_connected, _) = account_connected_reader();
     let share_link = {
         let username_or_principal = details.username_or_principal();
-        format!("/profile/{}?tab=tokens", username_or_principal)
+        format!("/profile/{}/posts", username_or_principal)
```

Comment:
posts or tokens?

Reply:
It's tokens we are keeping it the default route. Also that's anyways gonna be changed in the next PR of public wallet so I didn't touch it